code/datums/skills.dm

```diff
@@ -446,4 +446,88 @@
 	heavy_weapons = SKILL_HEAVY_WEAPONS_TRAINED
 	police = SKILL_POLICE_MP
 	powerloader = SKILL_POWERLOADER_MASTER
-	large_vehicle = SKILL_LARGE_VEHICLE_TRAINED
\ No newline at end of file
+	large_vehicle = SKILL_LARGE_VEHICLE_TRAINED
+
+//======//I.o.M.\\======\\
+
+/datum/skills/imperial
+	name = "Guardsman"
+	//cqc = SKILL_CQC_DEFAULT
```

Comment:
If you want the guardsmen to have zero skills, uncomment this all. Else select their desired skills.